numpy/core/defchararray.py

```diff
@@ -1797,6 +1817,8 @@ def isdecimal(a):
     For each element, return True if there are only decimal
     characters in the element.
 
+    Decimal characters are those that can be used to form numbers in base 10.
+
```

Comment:
I agree with @tylerjereddy , let's not duplicate the explanatory text:

```suggestion
```

Reply:
Removed!